frontEnd/tickets-app/src/components/Header/UserLogin.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { USER } from "../../store/user/selectors/selectors";
+import * as actions from "../../store/user/actions/actions";
+import { Link } from "react-router-dom";
+
+const UserLogin = () => {
+  const user = useSelector(USER);
+  const dispatch = useDispatch();
+  return (
+    <div
+      className="user"
+      onClick={user ? () => dispatch(actions.logout()) : null}
+    >
+      {user ? (
+        <React.Fragment>
```

Comment:
`React.Fragment` is `<> </>`, use the last one if you don't have attributes to implement